docs/building-a-form/available-components.md

```diff
@@ -348,18 +414,25 @@ When you build a form with more than one chapter (shown by the segments in a pro
 
 #### Usage guidelines
 
-Break into separate topic?
+In general, the review page renders the form data in review mode automatically. There are, however, some specific options you can pass to the form config to customize certain review functionality.
+
+This property is nested directly under `uiSchema`:
+- `'ui:reviewWidget'`: takes a widget component to render on the review page for that field. Default review widgets are automatically rendered, so only use this if you need to customize the review widget that is used.
+
+These properties are nested under `uiSchema: { `ui:options`: {} }`:
+- `hideOnReview`: Hides the specified field on the review page; takes a `boolean`
+- `hideOnReviewIfFalse`: Hides the specified field on the review page when the field value is `false`; takes a `boolean`
```

Comment:
Is "the specified field" the same field as "the field value"? For example if I have a checkbox with a title on it, does this hide the title in the review page unless it's checked?

Reply:
Sorry no, when I said "specified field" I did not mean the "field value", but the field itself. Yes, if this property is passed as `true`, it hides the entire field from the review page.

Here's what the review page would look like without passing that property:
<img width="728" alt="screen shot 2018-07-19 at 9 40 02 am" src="https://user-images.githubusercontent.com/25183456/42945945-bb7ecbe8-8b37-11e8-8528-8ceb8f2067e1.png">

Here's what the review page looks like when that property is passed:
<img width="676" alt="screen shot 2018-07-19 at 9 38 46 am" src="https://user-images.githubusercontent.com/25183456/42945906-9d3279e6-8b37-11e8-930a-4e828a271db8.png">

Do you think there's a way to make this wording more clear?

Reply:
Hah, the way I described it was the way you show it working. So my description doesn't work either. 😄 In simple words, I think the goal of this setting is to hide the title line on a checkbox unless it's checked, right?

I think what's confusing things is that the property itself takes a `true`/`false` but the `...IfFalse` in the name is referring to the `false` value of the field being described in the `uiSchema`. Maybe @bernars-usa has some ideas?

Reply:
As a group, we've been discussing `hideOnReviewIfFalse`. The behavior is extremely difficult to explain (especially because you can set `hideOnReviewIfFalse` to `true`! MY EYES! 😭).

We're going to open an issue to figure out what's happening here and possibly create a better implementation. In the meantime, I think the description that's here is ok, especially since people are unlikely to need this property in the short term.

Reply:
I opened an issue for this in https://github.com/usds/us-forms-system/issues/179.